LilitYeghiazaryan/file_words

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <map>
+
+std::string transform_word(std::string s)
+{
+    for (int i = 0; i < s.size(); i++) {
+        s[i] = tolower(s[i]);
+        if (s[i] < 'a' || s[i] > 'z')
+        {
+            s.erase(i, 1);
+            i--;
+        }
+    }
+    return s;
+}
+int main()
+{
+    int quantity;
+    std::cout << "Insert quantity of most common words: ";
+    std::cin >> quantity;
+
+    std::string w, filename, line;
+    std::fstream file;
+    filename = "war+peace.txt";
+    file.open(filename.c_str(),std::fstream::in | std::fstream::out | std::fstream::app);
```

Comment:
Քեզ ֆայլի մեջ գրել պետք չի այնպես որ բավարար է նաև ՝std::fstream::in՝ որը կարող ես նույնսիկ բաց թողնել քանի որ default-ով հենց դա է